org.scala-ide.sdt.core/src/org/scalaide/core/internal/async/AsyncDebuggerTab.scala

```diff
@@ -0,0 +1,123 @@
+package org.scalaide.core.internal.async
+
+import scala.util.Try
+
+import org.eclipse.core.runtime.CoreException
+import org.eclipse.debug.core.ILaunchConfiguration
+import org.eclipse.debug.core.ILaunchConfigurationWorkingCopy
+import org.eclipse.debug.internal.ui.SWTFactory
+import org.eclipse.debug.ui.AbstractLaunchConfigurationTab
+import org.eclipse.jface.dialogs.Dialog
+import org.eclipse.pde.internal.ui.IHelpContextIds
+import org.eclipse.swt.SWT
+import org.eclipse.swt.events.SelectionAdapter
+import org.eclipse.swt.events.SelectionEvent
+import org.eclipse.swt.graphics.Image
+import org.eclipse.swt.layout.GridData
+import org.eclipse.swt.layout.GridLayout
+import org.eclipse.swt.widgets.Button
+import org.eclipse.swt.widgets.Composite
+import org.eclipse.ui.PlatformUI
+import org.scalaide.ui.ScalaImages
+
+import AsyncDebuggerConfiguration.LaunchWithAsyncDebugger
+
+private class GeneralSettingsBlock(val tab: AsyncDebuggerTab) {
+  import AsyncDebuggerConfiguration._
+
+  private var globalAsyncDebuggerSwitchButton: Button = _
+  val SettingsArea = "General Settings"
+  val AsyncDebuggerGlobalSwitch = "Switch On/Off Async Debugger functionality for this debug configuration"
```

Comment:
I think "on off" is redundant when there's a check box right next to it. How about "Enable Async Debugger functionality".